src/seedu/addressbook/commands/FindCommand.java

```diff
@@ -50,8 +50,15 @@ public CommandResult execute() {
         final List<ReadOnlyPerson> matchedPersons = new ArrayList<>();
         for (ReadOnlyPerson person : addressBook.getAllPersons()) {
             final Set<String> wordsInName = new HashSet<>(person.getName().getWordsInName());
-            if (!Collections.disjoint(wordsInName, keywords)) {
-                matchedPersons.add(person);
+            // Consider matched even if case is inconsistent
+            for (String keyword: keywords) {
+                for (String word: wordsInName) {
```

Comment:
I noticed wordsInName is a HashSet, would it be better to do a .get() instead of looping through all the elements to take advantage of the HashSet's properties?